src/dev/build/lib/get_build_number.test.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+import { getBuildNumber } from './get_build_number';
+
+const BUILD_NUMBER_ENV_KEY = 'BUILD_NUMBER';
+
+let previousBuildNumber: string;
+
+beforeEach(() => {
+  if (BUILD_NUMBER_ENV_KEY in process.env) {
+    previousBuildNumber = process.env[BUILD_NUMBER_ENV_KEY] as string;
+    delete process.env[BUILD_NUMBER_ENV_KEY];
+  }
+});
+
+afterEach(() => {
+  if (BUILD_NUMBER_ENV_KEY in process.env) {
+    process.env[BUILD_NUMBER_ENV_KEY] = previousBuildNumber;
+  }
+});
+
+describe('getBuildNumber', () => {
+  it('returns env BUILD_NUMBER count', async () => {
+    process.env.BUILD_NUMBER = '123';
+    const buildNumber = await getBuildNumber();
+    expect(buildNumber).toBe(123);
+  });
+
+  it('returns git commit count', async () => {
+    const buildNumber = await getBuildNumber();
```

Comment:
It looks like this is still returning `1`.

Reply:
TIL: I guess github does the same thing for PRs as the build repo it only gets 1 commit will pass a build number to the CI. Probably why there wasn't any unit tests for this file.

Reply:
Resolved.

Reply:
Dang, do you think there's a way to change the command we use to fix that?